mlrun/projects/project.py

```diff
@@ -1480,7 +1695,7 @@ def __init__(
         self.conda = conda or {}
         self._mountdir = None
         self._artifact_mngr = None
-        self.artifact_path = artifact_path or config.artifact_path
+        self.artifact_path = artifact_path
```

Comment:
Why ?

Reply:
we dont need to save the default config, func run would do or with default config, if the user didnt specify a path it should stay blank and will get the default at runtime not at save time